### DISCUSSION TITLE
Translate after transcribe to transcribe only once

### DISCUSSION THREAD

**Discussion by DGrv** [openai/whisper#803] [Q&A]
Hi,

Is it possible to transcribe first and then do the translation ?
I think the translate is made via the model during the transcribe and not afterwards but it may already exists something.
The goal is to first transcribe and create subtitles in the original language and also translated.

If I do in python:

```py
model1 = whisper.load_model(options.modelwhisper)
result2 = model1.transcribe('input.mp3', language='fr')
result2["text"]
```

I would like to translate `result2` and then export it to srt (I do it with [stable-ts](https://github.com/jianfch/stable-ts))

**Comment by alkeryn** [comment]
@DGrv I don't know how you could do with that model, but you may tray with models like bloomz, T5 or mt0 that are on huggingface.
if you have a separate model specifically made for translation of text, you may have better result with the added bonus that you are not limited to translate only to english.
